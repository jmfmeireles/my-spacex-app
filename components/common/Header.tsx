import { useContext } from 'react';

import TranslationContext from '../../context/Translation';
import styles from '../../styles/Header.module.css';

const Header = () => {
  const { t } = useContext(TranslationContext);
  return (
    <header className={styles.header}>
      <h3 className={styles.title}>{t.title}</h3>
      <p className={styles.decription}>{t.description}</p>
    </header>
  );
};

export default Header;
