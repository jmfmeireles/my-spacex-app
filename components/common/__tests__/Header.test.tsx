import { render } from '@testing-library/react';
import Header from '../Header';

it('renders header properly', () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
