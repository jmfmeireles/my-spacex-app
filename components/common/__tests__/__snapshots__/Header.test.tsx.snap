// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header properly 1`] = `
<div>
  <header
    class="header"
  >
    <h3
      class="title"
    >
      My Space X App
    </h3>
    <p
      class="decription"
    >
      Next.js app that displays last Space X launches
    </p>
  </header>
</div>
`;
